src/tracing/librados.tp

```diff
@@ -2327,25 +2329,27 @@ TRACEPOINT_EVENT(librados, rados_watch2_exit,
     )
 )
 
-TRACEPOINT_EVENT(librados, rados_aio_watch_enter,
+TRACEPOINT_EVENT(librados, rados_aio_watch2_enter,
     TP_ARGS(
         rados_ioctx_t, ioctx,
         const char*, oid,
         rados_completion_t, completion,
         uint64_t*, phandle,
         rados_watchcb2_t, callback,
+        uint32_t timeout,
```

Comment:
@ryneli you missed a `,`. see http://gitbuilder.sepia.ceph.com/gitbuilder-ceph-deb-trusty-amd64-notcmalloc/log.cgi?log=29c70f92ec89d00399952d13145c9fe55daa7a01.